### PR TITLE
[Mime] Add "mime.mime_type_guesser" tag documentation

### PR DESCRIPTION
**Pull request by jschaedl**

### CODE REVIEW

File: reference/dic_tags.rst [hunk 1]
```diff
@@ -466,6 +467,22 @@ reuse the Symfony application between requests to improve performance. This tag
 is applied for example to the built-in :doc:`data collectors </profiler/data_collector>`
 of the profiler to delete all their information.
 
+.. _dic_tags-mime:
+
+mime.mime_type_guesser
+-----------------
```
Comment (OskarStark) [inline reference/dic_tags.rst]: The underline is to short

**Review by OskarStark — CHANGES_REQUESTED** [review]

File: reference/dic_tags.rst (hunk 1 above)
Comment (jschaedl) [inline reference/dic_tags.rst]:
```suggestion
----------------------
```

Comment (jschaedl) [reply reference/dic_tags.rst]: fixed

**Comment by javiereguiluz** [comment]
Jan, thanks for this contribution! We reworded it a bit while merging (see edf6e5e61f3b5c0efc19d9bbee6fd2b08b5f790b) to avoid repeating some of the internal details of guessing MIME types which is already mentioned in the main Mime component docs.

**Comment by jschaedl** [comment]
Thanks @javiereguiluz

**Pull request merged**